boards/arm/mps2_an521/CMakeLists.txt

```diff
@@ -1,9 +1,85 @@
 #
-# Copyright (c) 2019 Linaro Limited
+# Copyright (c) 2019,2020 Linaro Limited
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
 zephyr_library()
 zephyr_library_sources(pinmux.c)
 zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+
+if (CONFIG_BUILD_WITH_TFM)
+	# Set default image versions if not defined elsewhere
+	if (NOT DEFINED TFM_IMAGE_VERSION_S)
+		set(TFM_IMAGE_VERSION_S 0.0.0+0)
+	endif()
+
+	if (NOT DEFINED TFM_IMAGE_VERSION_NS)
+		set(TFM_IMAGE_VERSION_NS 0.0.0+0)
+	endif()
+
+	set(PREPROCESSED_FILE "${CMAKE_BINARY_DIR}/tfm/image_macros_preprocessed")
+	set(TFM_MCUBOOT_DIR "$ENV{ZEPHYR_BASE}/../modules/tee/tfm/trusted-firmware-m/bl2/ext/mcuboot")
```

Comment:
$ENV{ZEPHYR_BASE} is not mandatory, so this might fail.

Use ${ZEPHYR_BASE} instead.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/19985/commits/62b2df85cbd669ba451693bcb731137af4140e90